CONTRIBUTING.md

```diff
@@ -153,6 +154,15 @@ For overrides of virtual methods the `copydoc` tag can be used:
 
 A good example for code formatting and doxygen structure can be found in [swe_docu_guidelines.md (WIP)](./doc/aspice_swe3_4/swe_docu_guidelines.md)
 
+### External dependencies
+
+External dependencies such as the [STL](https://en.wikipedia.org/wiki/Standard_Template_Library) or
+other libaries shall be kept to a minium for `iceoryx_posh` and `iceoryx_hoofs`. If you think a new dependency is
+necessary, do the following:
+
+1. Engage with the maintainers beforehand by opening an issue to discuss the necessity
+1. If accepted, add the new header to `tools/scripts/used-headers.txt` for the CI to pass
```

Comment:
Please write 2. so that is also readable in my favorite editor vim

Reply:
There is no markdown plugin for vim? :astonished:

Reply:
Actually there is but I consider this bad style. In my opinion it is a feature of markdown that all the files are also readable in plain ASCII.

But as you can see during review this is not automatically converted so why not just enumerate it correctly.

Reply:
Well, three foods for thought:

1. It's readable in plain text
1. I'm very lazy
1. Maintainabilty is easier when moving lines around in a larger list

Would you install a markdown plugin for me or replace the numbers subsequently to right ones in your mind? :innocent: